Automation.sc

```diff
@@ -124,7 +125,7 @@ Automation {
      *      If win==nil, these bounds give the size of the new window.
      *      If both win and bounds are nil, defaults are used.
      */
-    front { |win=nil, bounds=nil|
+    front { |win=nil, bounds=nil| 
```

Comment:
Don't add a space at the end of the line. Also this is entirely unrelated to the patch.